packages/template-ui/src/components/ChannelHeader.vue

```diff
@@ -60,6 +91,12 @@ export default {
   padding-top: $navbar-height;
   height: $header-height;
```

Comment:
We should swap this `height` for `min-height` now that we're allowing the jumbotron to overflow. There's no guarantee that 5 lines of text will actually fit in $header-height, particularly as the title could wrap over two lines as well. For instance, the collapsed text will still overflow if the browser has a larger default font size.